netbout-web/src/test/java/com/netbout/rest/bout/XeMessageTest.java

```diff
@@ -70,6 +69,7 @@ public String author() {
                     }
                 }
             ).toXembly()
+        // @checkstyle IndentationCheck (1 line)
```

Comment:
@mkordas That is not a workaround. qulice requires to insert 4 spaces between the next closing bracket and the dot.